src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProviderTests.cs

```diff
@@ -30,16 +30,22 @@ interface IGoo
     void Goo();
     void Goo(int x);
     int Prop { get; }
+    int Generic<K, V>(K key, V value);
```

Comment:
Also add tests for escaped identifiers?

Reply:
i can add tests, but i'm certain that this feature currently screws that up.  I find myself not caring since teh chance of having an interface with a member name that is also a C# keyword is practically nil.   :)

Reply:
skipping this for now.  i just don't think the value is there.  I'm honestly trying to timebox this, not fully fix thsi feature to work properly for all cases.